src/View.elm

```diff
@@ -636,20 +640,47 @@ draftView { draft, currentUser } =
                                 else
                                     "Status"
                             ]
-                        , textarea
-                            [ id "status"
-                            , class "form-control"
-                            , rows 8
-                            , placeholder <|
-                                if hasSpoiler then
-                                    "This text will be hidden by default, as you have enabled a spoiler."
-                                else
-                                    "Once upon a time..."
-                            , onInput <| DraftEvent << UpdateStatus
-                            , required True
-                            , value draft.status
-                            ]
-                            []
+                        , let
+                            dec =
+                                (Decode.map
+                                    (\code ->
+                                        if code == 38 || code == 40 then
+                                            Ok NoOp
+                                        else
+                                            Err "not handling that key"
+                                    )
+                                    keyCode
+                                )
+                                    |> Decode.andThen fromResult
+
+                            options =
+                                { preventDefault = True, stopPropagation = False }
+
+                            fromResult : Result String a -> Decode.Decoder a
+                            fromResult result =
+                                case result of
+                                    Ok val ->
+                                        Decode.succeed val
+
+                                    Err reason ->
+                                        Decode.fail reason
+                          in
+                            textarea
+                                [ id "status"
+                                , class "form-control"
+                                , rows 8
+                                , placeholder <|
+                                    if hasSpoiler then
+                                        "This text will be hidden by default, as you have enabled a spoiler."
+                                    else
+                                        "Once upon a time..."
+                                , required True
+                                , onInputInformation <| DraftEvent << UpdateInputInformation
+                                , onClickInformation <| DraftEvent << UpdateInputInformation
+                                , property "defaultValue" (Encode.string draft.status)
+                                , onWithOptions "keydown" options dec
+                                ]
+                                []
```

Comment:
Note: at this point it feels slightly urgent to split the `View.elm` file, at least extracting all draft related views into it.